Source/UltraleapTrackingCore/Private/LeapWrapper.cpp

```diff
@@ -22,6 +23,22 @@ FLeapWrapper::FLeapWrapper()
 	, InterpolatedFrameSize(0)
 {
 	UseOpenXR = true;
+
+	if (!HaseDeactivateHandle.IsValid())
+	{
+		HaseDeactivateHandle = FCoreDelegates::ApplicationWillDeactivateDelegate.AddRaw(this, &FLeapWrapper::HandleApplicationDeactivate);
+	}
+}
+
+void FLeapWrapper::HandleApplicationDeactivate()
+{
+	// This will reset the tracking service for Android app, when the device goes to sleep 
+	AsyncTask(ENamedThreads::GameThread,
+		[this]()
+		{
+			ULeapBlueprintFunctionLibrary::UnbindTrackingServiceAndroid();
+			ULeapBlueprintFunctionLibrary::BindTrackingServiceAndroid();
```

Comment:
So we are forcing a reset of the tracking binding to get around the issue? Tracking SW suggested that we don't need to unbind when an application loses focus. I guess forcing an unbind then rebind is OK?

Reply:
I tried calling bind, but it didn't work, according to the guys from tracking, unbind then bind will reset it